client/state/sites/selectors/can-jetpack-site-auto-update-files.js

```diff
@@ -30,7 +25,7 @@ export default function canJetpackSiteAutoUpdateFiles( state, siteId ) {
 
 	const fileModDisabled = getSiteOption( state, siteId, 'file_mod_disabled' );
 
-	if ( fileModDisabled && includes( fileModDisabled, 'automatic_updater_disabled' ) ) {
+	if ( fileModDisabled && fileModDisabled.includes( 'automatic_updater_disabled' ) ) {
```

Comment:
Maybe we can just optional chaining and remove the `fileModDisabled &&`?

Reply:
I tried it and it crashed. Because `fileModDisabled` can be `false`.